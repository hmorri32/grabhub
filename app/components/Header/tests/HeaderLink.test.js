import React from 'react';
// import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { BrowserRouter as Router } from 'react-router-dom';

import HeaderLink from '../HeaderLink';

describe('<HeaderLink/>', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer
      .create(
        <Router>
          <HeaderLink to="whatever" />
        </Router>,
      )
      .toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
