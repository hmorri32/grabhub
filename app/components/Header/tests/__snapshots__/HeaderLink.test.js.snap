// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderLink/> should match the snapshot 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 0.25em 2em;
  margin: 0;
  -webkit-text-decoration: none;
  text-decoration: none;
  border-radius: 4px;
  -webkit-font-smoothing: antialiased;
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
  outline: 0;
  font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif;
  font-weight: bold;
  font-size: 16px;
  border: 2px solid #a76664;
  color: #a76664;
}

.c0:hover {
  border-color: #faa353;
  color: #faa353;
}

.c0:active {
  background: #faa353;
  color: #fff;
}

<a
  className="c0"
  href="/whatever"
  onClick={[Function]}
/>
`;
